### PR TITLE
Update crash spans' names and component

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
In accordance with the latest RUM backend changes

### CODE REVIEW

File: splunk-otel-android/src/main/java/com/splunk/rum/CrashReporter.java [hunk 1]
```diff
@@ -50,12 +50,13 @@ static void initializeCrashReporting(Tracer tracer, OpenTelemetrySdk openTelemet
...
             e.printStackTrace(new PrintWriter(writer));
-            tracer.spanBuilder("crash.report")
+            String exceptionType = e.getClass().getSimpleName();
+            tracer.spanBuilder(exceptionType)
                     .setAttribute(SemanticAttributes.THREAD_ID, t.getId())
                     .setAttribute(SemanticAttributes.THREAD_NAME, t.getName())
                     .setAttribute(SemanticAttributes.EXCEPTION_STACKTRACE, writer.toString())
                     .setAttribute(SemanticAttributes.EXCEPTION_ESCAPED, true)
-                    .setAttribute(SplunkRum.COMPONENT_KEY, SplunkRum.COMPONENT_ERROR)
+                    .setAttribute(SplunkRum.COMPONENT_KEY, SplunkRum.COMPONENT_CRASH)
```
Comment (breedx-splk) [inline splunk-otel-android/src/main/java/com/splunk/rum/CrashReporter.java:59]: I notice that `COMPONENT_ERROR` is still used for the ANRs...is that intentional?

**Review by breedx-splk — APPROVED** [review]
Looks good. Seems like maybe the only remaining usages of `COMPONENT_ERROR` are in the ANR code, so we should confirm that it's still wanted there and needed, otherwise remove it (future PR).

**Comment by mateuszrzeszutek** [comment]
> Looks good. Seems like maybe the only remaining usages of COMPONENT_ERROR are in the ANR code, so we should confirm that it's still wanted there and needed, otherwise remove it (future PR).

I'm going to submit a separate PR that adds setting `component=error` to `SplunkRum#addRumException()`, because that's what the RUM team expects (I think). But I'll wait for John with merging that PR, I'd like to get his opinion on this.

File: splunk-otel-android/src/main/java/com/splunk/rum/CrashReporter.java (hunk 1 above)
Comment (mateuszrzeszutek) [reply splunk-otel-android/src/main/java/com/splunk/rum/CrashReporter.java:59]: Hmm, I guess so? ANRs are not necessarily fatal, unrecoverable errors, it's just the application being horribly slow.
(At the very least, that's my understanding - and I don't know that much about Android 😅 )

**Pull request merged**